src/components/Profile/LikedMovies.js

```diff
@@ -0,0 +1,52 @@
+import React, { Fragment } from 'react';
+import { Link, useLocation } from 'react-router-dom';
+import { ChevronLeft } from '../styleElements/icons';
+import { ChevronRight } from '../styleElements/icons';
+import { CirclesBackground } from '../styleElements/CirclesBackground';
+import { dummyData } from '../MatchPage/MatchCard/dummyData';
+import { LogoActive } from '../styleElements/icons';
+
+export function LikedMovies() {
+	const location = useLocation();
+
+	const likedFilmsAll = [];
+	for (let i = 0; i < 8; i++) {
+		likedFilmsAll.push(dummyData[i]);
+	}
+
+	return (
+		<div className='liked_movies__container'>
+			<CirclesBackground />
+
+			<div className='profile__edit-footer'>
+				<div className='profile__chevron-left'>
+					<Link to='/Profile'>
+						<ChevronLeft size={30} active={location.pathname === '/Profile'} />
+					</Link>
+				</div>
+				<h1> Liked Movies </h1>
+			</div>
+
+			<div className='liked_movies__all'>
+				{likedFilmsAll.map((film) => (
```

Comment:
You're creating a list here so I would recommend assigning a Key to each entry.  It help performance in the long-run a ton.  I'm pretty sure all you need to do is:

<Fragment key={film.id}>

Reply:
This goes for everywhere you create a list like this.  I... should probably read up a bit more on exactly why it is important to assign a unique key to every entry in a list when using react but... I do know that it's important.  Let me get back to you on the reason😅